api/src/test/java/com/messagebird/TestUtil.java

```diff
@@ -146,6 +146,32 @@ static ContactList createContactList() {
         return contactList;
     }
 
+    private static VoiceStepOption createVoiceStepOption()
+    {
+        final VoiceStepOption voiceStepOption = new VoiceStepOption();
+        voiceStepOption.setDestination("31612345678");
```

Comment:
In my opinion, all properties on VoiceStepOtion class should be set with test values. In the unit tests, all of them should be checked with the assertion.

Reply:
I have just seen that you set all values in VoiceCallFlowTest class.  Nice. You can move those code in TestUtil class and can import it from here.